submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/FileListValidationTest.kt

```diff
@@ -72,13 +82,94 @@ class FileListValidationTest(
 
     @Test
     fun `valid file list`() {
-        val fileListFile = tempFolder.createFile("Plate1.tif")
-        val fileList = tempFolder.createFile("FileList.json", getFileListContent().toString())
+        val previousVersion = tsv {
+            line("Submission", "S-FLV123")
+            line()
 
-        webClient.uploadFiles(listOf(fileListFile, fileList))
-        webClient.validateFileList(fileList.name)
+            line("Study")
+            line()
 
-        webClient.deleteFile(fileListFile.name)
+            line("File", "Plate2.tif")
+            line()
+        }.toString()
+
+        val file1 = tempFolder.createFile("Plate1.tif", "content-1")
+        val file2 = tempFolder.createFile("Plate2.tif", "content-2")
+        val fileListContent = tsv {
+            line("Files", "Resource")
+            line("Plate1.tif", "USER_SPACE")
```

Comment:
what is the point of USER_SPACE here? SUBMISSION? this looks like a feature of the system or something like that when is a simple attribute. 

(please consider same comment for other test)